conf/modules/markduplicates.config

```diff
@@ -53,10 +53,10 @@ process {
         ext.prefix       = { "${meta.id}.sorted" }
         publishDir       = [
             // Never publish if BAM only should be published
-            enabled: !params.save_output_as_bam,
```

Comment:
why remove the enabled?

Reply:
not working in a closure, so it's now converted to a saveAs

Reply:
ah ok